test/colony-network-mining.js

```diff
@@ -91,6 +92,8 @@ contract("ColonyNetworkMining", accounts => {
     metaColony = await IMetaColony.at(metaColonyAddress);
     const clnyAddress = await metaColony.getToken();
     clny = await Token.at(clnyAddress);
+    goodClient = new ReputationMiner({ loader: contractLoader, minerAddress: MAIN_ACCOUNT, realProviderPort: REAL_PROVIDER_PORT, useJsTree });
```

Comment:
The instantiation or the `goodClient` is in the `beforeEach` no?

Reply:
Yup! ~Arguably~ we could instantiate clients in the tests only when they are needed (since we occasionally overwrite the `badClients`, but one thing at a time.